oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -151,6 +155,44 @@ public AccessToken refreshAccessToken() throws IOException {
     return new AccessToken(accessToken, new Date(expiresAtMilliseconds));
   }
 
+  /**
+   * Returns an Google Id Token from the metadata server on ComputeEngine.
+   * 
+   * @param targetAudience The aud: field the IdToken should include.
```

Comment:
nit: per Javadoc guidelines from Oracle, no initial caps (The --> the) and no period at end unless there's more than one sentence

Reply:
ok, removing inital capitals